beacon/validator/src/test/java/tech/pegasys/teku/validator/coordinator/publisher/AbstractBlockPublisherTest.java

```diff
@@ -50,19 +53,21 @@ public class AbstractBlockPublisherTest {
           new BlockPublisherTest(
               blockFactory, blockImportChannel, performanceTracker, dutyMetrics));
 
-  final SignedBlockContainer signedBlockContents = dataStructureUtil.randomSignedBlockContents();
+  final SignedBlockContents signedBlockContents = dataStructureUtil.randomSignedBlockContents();
+  final SignedBeaconBlock signedBlock = signedBlockContents.getSignedBlock();
 
   @BeforeEach
   public void setUp() {
-    when(blockFactory.unblindSignedBlockIfBlinded(signedBlockContents))
-        .thenReturn(SafeFuture.completedFuture(signedBlockContents));
+    when(blockFactory.unblindSignedBlockIfBlinded(signedBlock))
+        .thenReturn(SafeFuture.completedFuture(signedBlock));
   }
 
   @Test
   public void
       sendSignedBlock_shouldPublishImmediatelyAndImportWhenBroadcastValidationIsNotRequired() {
 
-    when(blockPublisher.importBlock(signedBlockContents, BroadcastValidationLevel.NOT_REQUIRED))
+    when(blockPublisher.importBlockAndBlobSidecars(
+            signedBlock, List.of(), BroadcastValidationLevel.NOT_REQUIRED))
```

Comment:
we are not testing that BlobSidecars are passed through since this change. Could we add BlobSidecars or add TODO or disable with TODO, so it is not lost in memories